packages/integrations/src/interfaces/indexer-manager/indexer.ts

```diff
@@ -0,0 +1,7 @@
+export interface Indexer {
+  id: number;
+  name: string;
+  url: string;
+  enable: boolean;
+  status: boolean;
```

Comment:
What is the difference between ``status`` and ``enable``?

Reply:
Enable: when the user enable / disable the indexer

Status: when there is an error with the indexer site.

On old homarr we did it the same.
If one of the options are false the indexer is off.

Reply:
Can you document this using JSDoc?
https://jsdoc.app/about-getting-started

Reply:
Yes